FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:238 in lock_acquire(): assertion `!lock_held_by_current_thread (lock)' failed.
Call stack: 0xc0120d18
Translation of call stack:
0xc0120d18: (unknown)
